stl/inc/atomic

```diff
@@ -230,7 +230,7 @@ _NODISCARD inline memory_order _Combine_cas_memory_orders(
 
     _Check_memory_order(_Success);
     _Check_load_memory_order(_Failure);
-    return _Combined_memory_orders[_Success][_Failure];
+    return _Combined_memory_orders[static_cast<int>(_Success)][static_cast<int>(_Failure)];

```

Comment:
This one is still `int`.  Should it also be changed to `unsigned int`?

Reply:
`int` is the underlying type, and we're assuming valid values, so it seems fine as-is. `unsigned int` wouldn't make invalid indices any safer.